### PR TITLE
apolo pharma

### PR DESCRIPTION
**Pull request by SRINIVAS383**
medicine

### CODE REVIEW

File: medicine [hunk 1]
```diff
@@ -0,0 +1,3 @@
+med 1 
+med 2
+med 3
```
Comment (vasu-devop) [inline medicine:3]: this source or formula is good but i need immediate results need some changes

**Comment by SRINIVAS383** [comment]
please check

**Comment by vasu-devop** [comment]
hiii

**Comment by SRINIVAS383** [comment]
yes please tell me

File: medicine (hunk 1 above)
Comment (vasu-devop) [reply medicine:3]: please chek and add med 5 also

**Pull request merged**